h/streamer/tweens.py

```diff
@@ -0,0 +1,19 @@
+__all__ = ["close_db_session_tween_factory"]
```

Comment:
Using `__all__` to explicitly declare the public interface of a module as well, as per PEP 8